locales/vi.schema.json

```
@@ -842,6 +871,12 @@
         },
         "paragraph": {
           "content": "Để thay đổi đoạn trích, hãy chỉnh sửa [bài viết blog].(https://help.shopify.com/en/manual/online-store/blogs/writing-blogs#display-an-excerpt-from-a-blog-post)"
```

Same here. Not part of this PR but I want to leave a comment because this will generate a broken link 
```suggestion
          "content": "Để thay đổi đoạn trích, hãy chỉnh sửa [bài viết blog](https://help.shopify.com/en/manual/online-store/blogs/writing-blogs#display-an-excerpt-from-a-blog-post)."
```

---

Fixed by @rafal-nedzarek-loc in the Smartling backend